Modular Scripts/File Monitoring/configure-file-monitoring.sh

```diff
@@ -2,7 +2,7 @@
 
 #downloads configure-linux.sh
 echo "INFO: Downloading dependencies - configure-linux.sh"
-curl -s -o configure-linux.sh https://www.loggly.com/install/configure-linux.sh
+curl -s -o configure-linux.sh https://raw.githubusercontent.com/Shwetajain148/install-script/tls-support-filemonitoring/Linux%20Script/configure-linux.sh
```

Comment:
@mchaudhary  @mostlyjason  For the time being, I have put the custom path here. After the changes will be deployed to the link [URL](https://www.loggly.com/install/configure-linux.sh), we can update it later.